### PR TITLE
Do Not Broadcast Attestations in Operations Service

### PR DESCRIPTION
**Pull request by rauljordan**
This PR stops broadcasting of attestations in the operations service and only broadcasts them once after receiving them via RPC

### CODE REVIEW

File: beacon-chain/rpc/attester_server.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/prysmaticlabs/prysm/shared/p2p"
+
```
Comment (terencechain) [inline beacon-chain/rpc/attester_server.go]:
```suggestion

```

File: beacon-chain/rpc/service.go [hunk 2]
```diff
@@ -8,6 +8,8 @@ import (
 	"net"
 	"time"
 
+	"github.com/prysmaticlabs/prysm/shared/p2p"
+
```
Comment (terencechain) [inline beacon-chain/rpc/service.go]:
```suggestion
```

File: beacon-chain/rpc/attester_server.go (hunk 1 above)
Comment (prestonvanloon) [inline beacon-chain/rpc/attester_server.go]: space

File: beacon-chain/rpc/service.go (hunk 2 above)
Comment (prestonvanloon) [inline beacon-chain/rpc/service.go]: space

**Pull request merged**